packages/expo-document-picker/android/src/main/java/expo/modules/documentpicker/DocumentPickerModule.kt

```diff
@@ -88,14 +89,19 @@ class DocumentPickerModule(
           }
         }
       }
-
       if (documentDetails == null) {
         promise.reject("E_DOCUMENT_PICKER", "Failed to read the selected document.")
       } else {
         val result = Bundle().apply {
           putString("type", "success")
           putString("uri", documentDetails.uri)
           putString("name", documentDetails.name)
+          putString("mimeType", documentDetails.mimeType)
+          if (documentDetails.mimeType == null) {
+            putParcelable("mimeType", null)
+          } else {
+            putString("mimeType", documentDetails.mimeType)
+          }
```

Comment:
```suggestion
		 documentDetails.mimeType?.let {
           putString("mimeType", documentDetails.mimeType)
		 }
```
You don't have to put null if the value isn't present. You can omit such value.